ott/geometry/pointcloud.py

```diff
@@ -184,7 +184,11 @@ def _compute_cost_matrix(self) -> jnp.ndarray:
     cost_matrix = self._cost_fn.all_pairs_pairwise(self.x, self.y)
     if self._axis_norm is not None:
       cost_matrix += self._norm_x[:, jnp.newaxis] + self._norm_y[jnp.newaxis, :]
-    return cost_matrix ** (0.5 * self.power)
+    cost_matrix = jax.lax.cond(
```

Comment:
I don't think we need the `lax.cond`, as `power` is a compile-time constant. Pure Python `if` should be sufficient.

Reply:
thanks!, learned something :)